src/main/java/games/strategy/engine/lobby/server/userDB/MutedMacController.java

```diff
@@ -124,39 +122,4 @@ public long getMacUnmuteTime(final String mac) {
     }
     return result;
   }
-
-  public List<String> getMacsThatAreStillMuted(final List<String> macs) {
```

Comment:
This method was introduced in 225dcdd, but there appears to never have been a call to it since.